dist/stencil-utils.min.js

```diff
@@ -1 +1 @@
d(parseFloat(e))}var r,o,i,a,s,u=pe.createElement("div"),c=pe.createElement("div");c.style&&(c.style.backgroundClip="content-box",c.cloneNode(!0).style.backgroundClip="",Te.clearCloneStyle="content-box"===c.style.backgroundClip,ke.extend(Te,{boxSizingReliable:function(){return e(),o},pixelBoxStyles:function(){return e(),a},pixelPosition:function(){return e(),r},reliableMarginLeft:function(){return e(),s},scrollboxSize:function(){return e(),i}}))}();var bt=["Webkit","Moz","ms"],wt=pe.createElement("div").style,xt={},_t=/^(none|table(?!-c[ea]).+)/,Tt=/^--/,jt={position:"absolute",visibility:"hidden",display:"block"},Ot={letterSpacing:"0",fontWeight:"400"};ke.extend({cssHooks:{opacity:{get:function(e,t){if(t){var n=F(e,"opacity");return""===n?"1":n}}}},cssNumber:{animationIterationCount:!0,columnCount:!0,fillOpacity:!0,flexGrow:!0,flexShrink:!0,fontWeight:!0,gridArea:!0,gridColumn:!0,gridColumnEnd:!0,gridColumnStart:!0,gridRow:!0,gridRowEnd:!0,gridRowStart:!0,lineHeight:!0,opacity:!0,order:!0,orphans:!0,widows:!0,zIndex:!0,zoom:!0},cssProps:{},style:function(e,t,n,r){if(e&&3!==e.nodeType&&8!==e.nodeType&&e.style){var o,i,a,s=m(t),u=Tt.test(t),c=e.style;if(u||(t=U(s)),a=ke.cssHooks[t]||ke.cssHooks[s],void 0===n)return a&&"get"in a&&void 0!==(o=a.get(e,!1,r))?o:c[t];i=typeof n,"string"===i&&(o=Ye.exec(n))&&o[1]&&(n=_(e,t,o),i="number"),null!=n&&n===n&&("number"!==i||u||(n+=o&&o[3]||(ke.cssNumber[s]?"":"px")),Te.clearCloneStyle||""!==n||0!==t.indexOf("background")||(c[t]="inherit"),a&&"set"in a&&void 0===(n=a.set(e,n,r))||(u?c.setProperty(t,n):c[t]=n))}},css:function(e,t,n,r){var o,i,a,s=m(t);return Tt.test(t)||(t=U(s)),a=ke.cssHooks[t]||ke.cssHooks[s],a&&"get"in a&&(o=a.get(e,!0,n)),void 0===o&&(o=F(e,t,r)),"normal"===o&&t in Ot&&(o=Ot[t]),""===n||n?(i=parseFloat(o),!0===n||isFinite(i)?i||0:o):o}}),ke.each(["height","width"],function(e,t){ke.cssHooks[t]={get:function(e,n,r){if(n)return!_t.test(ke.css(e,"display"))||e.getClientRects().length&&e.getBoundingClientRect().width?X(e,t,r):nt(e,jt,function(){return X(e,t,r)})},set:function(e,n,r){var o,i=gt(e),a=!Te.scrollboxSize()&&"absolute"===i.position,s=a||r,u=s&&"border-box"===ke.css(e,"boxSizing",!1,i),c=r?G(e,t,r,u,i):0;return u&&a&&(c-=Math.ceil(e["offset"+t[0].toUpperCase()+t.slice(1)]-parseFloat(i[t])-G(e,t,"border",!1,i)-.5)),c&&(o=Ye.exec(n))&&"px"!==(o[3]||"px")&&(e.style[t]=n,n=ke.css(e,t)),z(e,n,c)}}}),ke.cssHooks.marginLeft=B(Te.reliableMarginLeft,function(e,t){if(t)return(parseFloat(F(e,"marginLeft"))||e.getBoundingClientRect().left-nt(e,{marginLeft:0},function(){return e.getBoundingClientRect().left}))+"px"}),ke.each({margin:"",padding:"",border:"Width"},function(e,t){ke.cssHooks[e+t]={expand:function(n){for(var r=0,o={},i="string"==typeof n?n.split(" "):[n];r<4;r++)o[e+Je[r]+t]=i[r]||i[r-2]||i[0];return o}},"margin"!==e&&(ke.cssHooks[e+t].set=z)}),ke.fn.extend({css:function(e,t){return Fe(this,function(e,t,n){var r,o,i={},a=0;if(Array.isArray(t)){for(r=gt(e),o=t.length;a<o;a++)i[t[a]]=ke.c
```

Comment:
Let's not include changes to the dist file until we are ready to do a release.

Reply:
resetted dist file.